packages/neuron-wallet/src/utils/common.ts

```diff
@@ -10,13 +10,13 @@ export default class CommonUtils {
 
     while (++retryTime < times) {
       try {
-        return callback()
+        return await callback()
```

Comment:
`await` is redundant in `return statement`

Reply:
The exception can't be caught if the callback's result is a promise without await.

Reply:
gotcha